src/output/graphical_x11/main.c

```diff
@@ -172,6 +172,38 @@ void calculateColors(XAVA_CONFIG *conf) {
     }
     snatchColor("color5", conf->color, &conf->col, databaseName, &xavaXResDB);
     snatchColor("color4", conf->bcolor, &conf->bgcol, databaseName, &xavaXResDB);
+    int lineNumberFg = 1;
```

Comment:
Lets avoid having color logic inside of the output methods themselves, but rather move that somewhere where it could be reused easily, like one of the shared functions. This way we avoid bloating the modules themselves unnecessarily.

 ``src/shared/config/pywal`` sounds like a good place to start.